packages/renderer/src/lib/preferences/PreferencesConnectionDetailsTerminal.svelte

```diff
@@ -0,0 +1,187 @@
+<script lang="ts">
+import '@xterm/xterm/css/xterm.css';
+
+import type { ProviderConnectionShellDimensions, ProviderConnectionStatus } from '@podman-desktop/api';
+import { EmptyScreen } from '@podman-desktop/ui-svelte';
+import { FitAddon } from '@xterm/addon-fit';
+import { SerializeAddon } from '@xterm/addon-serialize';
+import { Terminal } from '@xterm/xterm';
+import { onDestroy, onMount } from 'svelte';
+import { router } from 'tinro';
+
+import { getExistingTerminal, registerTerminal } from '/@/stores/provider-terminal-store';
+import type { ProviderContainerConnectionInfo, ProviderInfo } from '/@api/provider-info';
+
+import { TerminalSettings } from '../../../../main/src/plugin/terminal-settings';
+import { getTerminalTheme } from '../../../../main/src/plugin/terminal-theme';
+import NoLogIcon from '../ui/NoLogIcon.svelte';
```

Comment:
nitpick : prefer the `@/` import rather than  `../`

Reply:
All of the files with the TerminalSettings/getTerminalTheme has this path :( I was not able to change it to `@/`

Reply:
`/@/lib/ui/NoLogIcon.svelte` ?

Reply:
Oh! I ment `../../../../main/src/plugin/terminal-settings` and the other one :D 
The NoLogIcon works :D